v22.1/add-constraint.md

```diff
@@ -19,6 +19,8 @@ The [`DEFAULT`](default-value.html) and [`NOT NULL`](not-null.html) constraints
 
 {% include {{ page.version.version }}/sql/combine-alter-table-commands.md %}
 
+We don't like this file bababooie asdlfkmflk
```

Comment:
Misspellings currently break the build. I get that Warnings are easy to ignore, but I'm worried that such a strict policy will cause more headaches than it solves for new terms we haven't yet added to the exception list.

Reply:
I'll send out a poll to the team to see what people would want to see in terms of functionality.

Reply:
I personally would prefer that avenue, and I can add documentation in the wiki as to how to edit the list of terms.